juju/juju
# [JUJU-1318] create new service locator collection in db

This PR adds a new collection in DB,  the collection so far contains only Service locator Name, Type, ID. 
Further, this collection will be used as 'build-in' assertions information storage and service locator data storage.

## QA steps

```sh
juju bootstrap localhost lxd
```
Take script to login to DB: https://discourse.charmhub.io/t/login-into-mongodb/309

login to DB and type: `show collections`

one of them should be `ServiceLocators`

## Documentation changes

None

## Bug reference

None
